web-service-consumer/1.6/modules/ROOT/pages/edits/web-service-consumer-studio.adoc

```diff
@@ -0,0 +1,282 @@
+
+Add Web Service Consumer Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type `web service consumer` in the search field.
+. Click *Web Service Consumer* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+
+[[configure-input-source]]
+== Configure an Input Source
+
+An input source initiates a flow when a specified condition is met.
+
+For example, to configure `HTTP Listener`, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP* > *Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a global element that can be used by all instances of `HTTP Listener` in the app.
+. On the *General* tab, specify connection information.
+. On the *TLS* tab, optionally specify TLS information.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specific server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add Consume Operation to the Flow
+
+When you add a connector operation to your flow, you immediately define a specific operation for that connector to perform.
+
+To add the Consume operation for Web Service Consumer Connector, follow these steps:
+
+. In the *Mule Palette* view, select *Web Service Consumer* and then select the *Consume* operation.
+. Drag the operation onto the Studio canvas to the right of the input source.
+
+In Studio, the Consume operation looks like this:
+
+.Web Service Consume Operation in Studio
+image::web-service-consumer-consume-operation.png[Web Service Consume Operation in Studio]
+
+[[configure-global-element]]
+== Configure a Global Element for the Connector
+
+When you configure a connector, it’s best to configure a global element that all instances of that connector in the app can use. To consume a SOAP Web service operation, you need to create a configuration that points to the service that you want to consume.
+
+To configure the global element for Web Service Consumer Connector, follow these steps:
+
+. Select the name of the connector in the Studio canvas.
+. In the *Basic Settings* section of the *General* tab, click the plus sign (+) next to the *Connector configuration* field to access the global element configuration fields.
+. On the *Connection* section of the *General* tab, configure the following parameters:
+* *WSDL location* +
+The WSDL file URL remote or local.
```

Comment:
DITTO rewording suggested in #923 
Also, remove periods, because these are not sentences.